### DISCUSSION TITLE
A way to make the canister shells performance a bit better.

### DISCUSSION THREAD

**Discussion by link12313** [FakeFishGames/Barotrauma#5417] [Unstable feedback]
You could make it stagger the projectile creation a little bit to improve performance. Maybe firing a 
 group of projectiles every other frame will work.

**Comment by Ek-Videogames** [comment]
Whatever method of updating them must be performance intensive - at least in its current 'newly added' implementation?
I've not usually had noticeable performance problems from spawning a handful of items at once in modding. If i had to guess this will probably get more optimized as it heads towards stable making this compromise not needed if all goes well.
